elyra/pipeline/parser.py

```diff
@@ -25,24 +25,30 @@ class PipelineParser(LoggingConfigurable):
 
     @staticmethod
     def parse(pipeline_definition) -> Pipeline:
-
-        # The pipeline blueprint enables defining multiple pipelines
-        # in one pipeline json file. For now we only support processing
-        # one (primary) pipeline.
+        """
+        The pipeline blueprint enables defining multiple pipelines
+        in one pipeline json file. When SuperNodes are enabled, each
+        super node references another flow/pipeline in the pipeline
+        blueprint. For now, we are not supporting SuperNodes and  we
+        are only support processing one (primary) pipeline.
+        """
 
         # Check for required values.  We require a primary_pipeline, a set of pipelines, and
         # nodes within the primary pipeline (checked below).
         if 'primary_pipeline' not in pipeline_definition:
             raise SyntaxError("Required field: 'primary_pipeline' not found.")
         if 'pipelines' not in pipeline_definition:
             raise SyntaxError("Required field: 'pipelines' not found.")
+        if len(pipeline_definition['pipelines']) > 0:
```

Comment:
Should this be `> 1`?  Perhaps this is why the tests are failing?

Is it possible to add a `test_no_supernodes()` test?  It would be something that is still useful when supernodes are supported, but disabled.